aiounifi/models/wlan.py

```diff
@@ -20,6 +20,7 @@ class TypedWlan(TypedDict):
     dtim_ng: int
     enabled: bool
     group_rekey: int
+    hide_ssid: bool
```

Comment:
```suggestion
    hide_ssid: NotRequired[bool]
```
You're checking for None so I would expect this is needed